### PR TITLE
Add language translation support & translations for Spanish (us) & Arabic

### PR DESCRIPTION
**Pull request by richard-decastongrene-electro**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     },
     output: {
         path: path.resolve(__dirname, 'dist'),
-        clean: true,
+        clean: false,
```
Comment (electroross) [inline webpack.config.js]: Why aren't we cleaning out the dist folder when it builds?

File: src/js/ot-dns-script-1.js [hunk 2]
```diff
@@ -83,6 +78,9 @@ async function OptanonWrapperLocal() {
     // Optanon.InsertScript('otDnsScript2.js', 'body', null, null, 'C0001', true);
```
Comment (electroross) [inline src/js/ot-dns-script-1.js]: Delete commented out code please

**Review by electroross — CHANGES_REQUESTED** [review]
Some minor items

File: CHANGELOG.md [hunk 3]
```diff
@@ -60,3 +60,6 @@
 ## 1.2.7 Fix regex pattern console error
 - HTML pattern attribute change to use regex v flag (needs more chars escaped in character classes)
 - https://groups.google.com/a/chromium.org/g/blink-dev/c/gIyvMw0n2qw
+
+## 1.3.0 Add support for Spanish UI
+- Placeholder text only
```
Comment (electroross) [inline CHANGELOG.md]: This is no longer true, I think

**Review by electroross — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (richard-decastongrene-electro) [reply webpack.config.js]: Fixed

File: CHANGELOG.md (hunk 3 above)
Comment (richard-decastongrene-electro) [reply CHANGELOG.md]: Updated changelog

**Pull request merged**